attestation-agent/kbs_protocol/src/lib.rs

```diff
@@ -17,7 +17,7 @@ pub mod types;
 const KBS_REQ_TIMEOUT_SEC: u64 = 60;
 const KBS_GET_RESOURCE_MAX_ATTEMPT: u64 = 3;
 
-pub const KBS_URL_PREFIX: &str = "kbs/v0";
+pub const KBS_URL_PREFIX: &str = "/kbs/v0";
```

Comment:
maybe `KBS_RESOURCE_PREFIX` is better?

Reply:
The prefix here is not only used to obtain resources, but also to obtain tokens in the passport model. Perhaps we can just call it `KBS_PREFIX`.